### PR TITLE
[docs] typos in deep-linking.mdx

### PR DESCRIPTION
**Pull request by rmschindler**

### CODE REVIEW

File: docs/pages/guides/deep-linking.mdx [hunk 1]
```diff
@@ -47,7 +47,7 @@ Build your native app with EAS Build to ensure the entitlement is registered wit
 
 <ConfigReactNative title="Manual native configuration">
 
-Apps that don't use [Continuous Native Generation](/workflow/continuous-native-generation) (`npx expo prebuild`) must [manually configure](/build-reference/ios-capabilities#manual-setup) the **Associated Domains** capability for their bundle identifier.
```
Comment (Simek) [inline docs/pages/guides/deep-linking.mdx:50]: This change does not look correct.

**Review by Simek — COMMENTED** [review]
Hello @rmschindler, thanks for fixing the typos! Left one suggestion, before we can ship it.

File: docs/pages/guides/deep-linking.mdx (hunk 1 above)
Comment (rmschindler) [reply docs/pages/guides/deep-linking.mdx:50]: you are right; I mistakenly made this change to the link

File: docs/pages/guides/deep-linking.mdx [hunk 2]
```diff
@@ -17,7 +17,7 @@ Before you can use app/universal links, you have to set up the two-way associati
 1. **Native app verification:** This requires some form of code signing that references the target website domain (URL).
 2. **Website verification:** This requires a file to be hosted on the target website in the **/.well-known** directory.
 
-After the two-way association is setup, you have to setup the runtime routing of the link in your app. This is done in JavaScript and must be configured for every route, then coordinated between web and native. Expo offers a fully automated solution for this called [Expo Router](/router/introduction/).
+After the two-way association is set up, you have to set up the runtime routing of the link in your app. This is done in JavaScript and must be configured for every route, then coordinated between web and native. Expo offers a fully automated solution for this called [Expo Router](/router/introduction/).
```
Comment (amandeepmittal) [inline docs/pages/guides/deep-linking.mdx:20]:
```suggestion
After setting up the two-way association, you have to set up the runtime routing of the link in your app. This is done in JavaScript and must be configured for every route, then coordinated between web and native. Expo offers a fully automated solution for this called [Expo Router](/router/introduction/).
```

Changes not related to this PR.

**Review by amandeepmittal — APPROVED** [review]
Thanks for fixing these typos @rmschindler 👍

**Pull request merged**